tests/acceptance/features/apiReshare/resharing.feature

```diff
@@ -140,4 +151,4 @@ Feature: re-share resources
       | 27          | share + view + edit + delete          |
       | 29          | share + view + create + delete        |
       | 31          | share + view + create + edit + delete |
-      
+
```

Comment:
```suggestion
```

Reply:
this is good. we should keep the newline